### DISCUSSION TITLE
Is it possible to copy files between workspaces?

### DISCUSSION THREAD

**Discussion by Climenty** [leo-arch/clifm#307] [Q&A] [answered]
Subj!
Something like this: `c 2 @ws3`

**Comment by leo-arch** [comment, marked as answer]
Hi @Climenty. Currently, this is not possible. The way to do this is this:

```sh
s 2 # select the source file
Alt-3 # switch to the third workspace
c sel # copy the selected file into the current directory
```
However, workspace expansion is a good idea. Added to the TODO list (maybe for the next release).

**Reply by Climenty** [reply]
Thanks for the answer, the nuances of a new tool are not always clear.

**Comment by leo-arch** [comment]
Hi @Climenty. Since https://github.com/leo-arch/clifm/commit/8faa119a30b0aab1a8896029cb24dbc94fc1a3ee (version 1.20.9) you can use the `w:` construct to expand workspaces. For instance, using your example:

```
c 2 w:3
```
to copy the file whose ELN is 2 to the third workspace.
